### ISSUE TITLE
Choosing epochs based on test results?

### ISSUE THREAD

**Issue by thisishale** [ChuhuaW/SGNet.pytorch#24]
Hello,

Thank you very much for your contribution. I have a question about the validation and test process. From the code, it seems like the best epoch is chosen based on test loss, while it should be based on validation loss. 
Could you clarify?  as I am confused about this. Thanks.

**Comment by YuchenWang** [comment]
Hi,

It depends on which dataset you mentiond. For the ETH and UCY datasets, a leave-one-out strategy is used for evaluation, similar to previous works , where the model is trained on four datasets and evaluated on the held-out fifth. For the nuScenes dataset, we use the official dataset split for the contest.

**Comment by thisishale** [comment]
Thank you for your reply. My question was mainly about the PIE dataset.
